### DISCUSSION TITLE
Expose the current runner version as an environment variable

### DISCUSSION THREAD

**Discussion by tbutler-qontigo** [actions/virtual-environments#10207] [Ideas]
Can we have this:
![image](https://github.com/actions/runner-images/assets/72974415/49dc5463-7493-4a96-a2d1-7a01ed8ca1b4)
available as a standard environment variable?

My use case is that we are using a mix of github standard runners and self-hosted runners and we want to periodically check that our runners are "in sync" with the standard runners.  
We are checking certain tool versions like bash, pwsh, pester etc but we would also like to ensure we are using the same runner version too, so that we can minimise differences between standard and self-hosted runners and reduce the likelihood of getting different results depending on what type of runner was used.

Thanks

**Comment by erik-bershel** [comment]
Hey @tbutler-qontigo!

Architecturally, we cannot accomplish this on our end. Please, try to offer the same thing in the original repository, I suppose they will provide more informative reply. 
- https://github.com/actions/runner
